shared/chat/inbox/row/big-teams-divider/index.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import * as React from 'react'
+import {ClickableBox, Icon, Box, Badge} from '../../../../common-adapters'
+import {globalStyles, globalColors, globalMargins, glamorous, isMobile} from '../../../../styles'
+import {BigTeamsLabel} from '../big-teams-label'
+
+type Props = {
+  badgeCount: number,
+  toggle: () => void,
+}
+
+const DividerBox = glamorous(Box)({
```

Comment:
i thought we're avoiding glamorous?

Reply:
we are and i changed most of the hot things (rows which mutate and things we scroll around) this is basically a singleton which never changes so leaving it for now. Once chat is in i do want to spend time on making our styling story better so we'll do a larger cleanup then